layout/theme.liquid

```diff
@@ -36,6 +36,10 @@
       assign body_font_bold = settings.type_body_font | font_modify: 'weight', 'bold'
       assign body_font_italic = settings.type_body_font | font_modify: 'style', 'italic'
       assign body_font_bold_italic = body_font_bold | font_modify: 'style', 'italic'
+      assign page_width_margin = 0
+      if settings.page_width == '1600' 
+        assign page_width_margin = 2
```

Comment:
Yea, I am not sure 🤔 if you have a better idea for naming that would be great 😅 

I also haven't gotten this done yet: https://github.com/Shopify/dawn/pull/576#discussion_r704650941

We already have some negative margin complexity and some sections that have padding instead of margin to separate it so I will need to add the variable there as well. I wanted to get a design review first to ensure it looks good before making the changes everywhere